src/main/python/login.py

```diff
@@ -114,7 +114,7 @@ def attempt_login(self, username, pwd):
                 """
                 self.accept()
                 self.login_signal.emit()
-            elif status == 200 and res.get("auth") == False:
+            elif status == 200 and not res.get("auth"):
```

Comment:
In this case, it should better return `401(Unauthorized)` instead of `200`.

Reply:
Hmm. I'm not entirely sure what's going on here. The API doesn't make a distinction via HTTP return code as to whether login failed due to incorrect email or password (at least it shouldn't). Looking at the code in the API again just now, I'm thinking it should definitely be rewritten to be a bit more clear as to why a login call could have failed.

Reply:
Of course, just for now, it should be good to go, but from the long term perpertive, we will still need to have some optimize for the API to make the status code be more useful. It would be make more sense when we receive `200`, and we can assume everything goes well and login has been approved. Username incorrect might be either `204(no content)` or `404(Not Found)`, and password incorrect might be `401(Unauthorized)`.

Reply:
Agreed. But I think it should be taken care of in another PR.

Reply:
would this not be a fix on the server side?  make the api return the status code as well as error message, then render that client side?

Reply:
This would be a change in the API's code. I updated my last comment to make that point more explicit.